data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -167,40 +193,37 @@ def _climate_factor(self) -> bool:
             )
         )
 
-        return (
-            self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-                + field_names.PERCENTILE_FIELD_SUFFIX
-            ]
-            >= self.LOW_INCOME_THRESHOLD
-        ) & climate_criteria
+        return self.FPL_200_SERIES & climate_criteria
 
     def _energy_factor(self) -> bool:
         # In Xth percentile or above for DOE’s energy cost burden score (Source: LEAD Score)
         # AND
         # Low income: In 60th percentile or above for percent of block group population
         # of households where household income is less than or equal to twice the federal
-        # poverty level. Source: Census's American Community Survey]
-        energy_criteria = (
+        # poverty level. Source: Census's American Community Survey]  
+
+        threshold_one = (
             self.df[
                 field_names.ENERGY_BURDEN_FIELD
                 + field_names.PERCENTILE_FIELD_SUFFIX
             ]
             >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-        ) | (
+        ) 
+
+        threshold_two = (
             self.df[
                 field_names.PM25_FIELD + field_names.PERCENTILE_FIELD_SUFFIX
             ]
             >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-        )
+        ) 
 
-        return (
-            self.df[
-                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
-                + field_names.PERCENTILE_FIELD_SUFFIX
-            ]
-            >= self.LOW_INCOME_THRESHOLD
-        ) & energy_criteria
+        self.df[
+            field_names.ABOVE_90TH_FOR_COST_BURDEN_LOW_INCOME
+        ] = threshold_one & self.FPL_200_SERIES
+
+        self.df[field_names.PM25_LOW_INCOME] = threshold_two & self.FPL_200_SERIES
+
+        return (threshold_one | threshold_two) & self.FPL_200_SERIES
```

Comment:
Why not replace this with: 

```
return (field_names.ABOVE_90TH_FOR_COST_BURDEN_LOW_INCOME | self.df[field_names.PM25_LOW_INCOME])
``` 

That would reduce the duplicate logic of checking `threshold_one & self.FPL_200_SERIES`